documentation/principles/cost-effective.md

```diff
@@ -2,12 +2,17 @@
 layout: sub-navigation
 order: 105
 title: Cost Effective
-excerpt: Short extract
+excerpt: We aim to help teams reduce build and run costs.
 tags:
 - principle
 eleventyNavigation:
   parent: principles
 ---
-Core Cloud operates consumption-based billing meaning teams only pay for their usage.\
-This incentivises teams to take responsibility and optimise resources attributed to them.\
-Core Cloud supports teams in optimising their resources using techniques like autoscaling, downscaling, spot pricing and ARM powered instances.
\ No newline at end of file
+**We operate consumption-based billing**. \
+This means teams only pay for their usage and are incentivised to optimise their resources. 
```

Comment:
```suggestion
Customers are charged on a consumption model and are incentivised to optimise their resources. 
```
Suggesting `Customer` over team or account as they can mean different things to different people. Also changed from saying only pay for their usage because at some point I'm sure there will be overheads that need to be covered i.e. building in costs for the support model will need to be paid regardless of whether customers submit any support requests or not

Reply:
👍